lib/Versions/GroupVersion.php

```diff
@@ -46,12 +41,11 @@ public function __construct(
 		FileInfo $sourceFileInfo,
 		IVersionBackend $backend,
 		IUser $user,
-		File $versionFile,
-		int $folderId
+		string $label,
+		private File $versionFile,
+		private int $folderId,
 	) {
-		parent::__construct($timestamp, $revisionId, $name, $size, $mimetype, $path, $sourceFileInfo, $backend, $user);
-		$this->versionFile = $versionFile;
-		$this->folderId = $folderId;
+		parent::__construct($timestamp, $revisionId, $name, $size, $mimetype, $path, $sourceFileInfo, $backend, $user, $label);
```

Comment:
Both min and max version will need to be 28 then? Is that already the case in info.xml?